src/workerd/server/tests/container-client/BUILD.bazel

```diff
@@ -1,9 +1,75 @@
 load("//:build/wd_test.bzl", "wd_test")
 
+# TODO(soon): Replace this with https://github.com/bazelbuild/rules_docker
```

Comment:
This means we also have to install this dependency in downstream repo. @fhanau any recommendations on how to make the following ugly code better?

Reply:
Shouldn't using `off-by-default` be sufficient to not have this run downstream?

Reply:
Yes, the goal is to run this on GitHub workflow.